kinitro/api/__init__.py

```
@@ -0,0 +1,6 @@
+"""API service for Kinitro evaluation backend."""
+
+from kinitro.api.app import create_app, run_server
+from kinitro.api.config import APIConfig
+
+__all__ = ["create_app", "run_server", "APIConfig"]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)